docs/getting-started/quick_start.md

```diff
@@ -60,31 +92,76 @@ It should be in iterable, yielding tuples, of the same length as the number of a
 
 <!--pytest-codeblocks:skip-->
 ```python
-inputset = [(2, 3), (0, 0), (1, 6), (7, 7), (7, 1)]
+max_value_included = 7
+inputset = [
+    (
+        np.random.randint(max_value_included + 1),
+        np.random.randint(max_value_included + 1),
+    )
+    for _ in range(10)
+]
 ```
 
+Here, our inputset is made of 10 pairs of integers, whose maximum value is `max_value_included`. Remark that the values which will later be used as inputs of the compiled function need to be in the same range.
+
 {% hint style="warning" %}
-All inputs in the inputset will be evaluated in the graph, which takes time. If you're experiencing long compilation times, consider providing a smaller inputset.
+All inputs in the inputset will be evaluated in the graph, which takes time. If you're experiencing long compilation times, consider providing a smaller inputset. At the same time, a too-short inputset means that the bit widths which are measured during the tracing may not be exact, and the consequence would be that the function correctness is not ensured.
```

Comment:
Even with a big inputset, function correctness is not `ensured` and tracing is pretty much never `exact`. I think something like this would be better:
```
At the same time, if the inputset is too small, bit widths may not be measured accurately, which could result in overflows during runtime, hence the result wouldn't be what's expected.
```

Reply:
suggestion taken as well